### PR TITLE
Feature/new download page

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: src/templates/download-page.js [hunk 1]
```diff
@@ -105,72 +141,73 @@ class DownloadTemplate extends React.Component {
...
+                  </h1>
+                </div>
+                <div className="u-spaceMB5">
+                  <div className="u-textCenter">
+                    <div>
+                      {isSuccessful ? (
+                        <div>{successText}</div>
+                      ) : (
+                        <form onSubmit={this.handleSubmit}>
+                          <input
```
Comment (palmfjord) [inline src/templates/download-page.js]: Could we maybe break out some of these into styled components and stuff instead?

Comment (hedvigoscar) [reply src/templates/download-page.js]: Yes!

In fact I'm doing that right now, dont you worry :wink:

Comment (palmfjord) [reply src/templates/download-page.js]: Åhå! Nice :D

File: .prettierrc.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  semi: true,
+  semi: false,
```
Comment (palmfjord) [inline .prettierrc.js]: This is awesome!! But maybe we should do it in as separate PR? 🙈 because it's kind of affecting eeeeeeverything

**Comment by hedvigoscar** [comment]
Obsoleted by the clean PR. Closing
